### PR TITLE
Adds yaml filter to fix unicode text conversion in admin list

### PR DESCRIPTION
**Pull request by eemperor**

### CODE REVIEW

**Comment by lorengordon** [comment]
@eemperor Can you also test to be sure that this is compatible with salt 2016.11?

**Comment by eemperor** [comment]
Okay

**Comment by eemperor** [comment]
Change works fine with Salt 2016.11.  Have to see what the failed Travis check is all about.

**Comment by lorengordon** [comment]
The travis failures may be about how it is now picking up salt 2019.2.0...

**Comment by eemperor** [comment]
Yup.  In Salt 2019.2.0, it simply errors on the unicode text instead of just passing it along with the `u` prefix as previous versions.

https://docs.saltstack.com/en/latest/topics/releases/2019.2.0.html

I will need to run tests on Linux to verify this change.

**Comment by eemperor** [comment]
Tests with Salt 2016.11.9 and 2018.3.3 on CentOS and RedHat were all successful.

@plus3it/salt-formulas ready for review

File: join-domain/windows/init.sls [hunk 1]
```diff
@@ -15,7 +15,7 @@ join standalone system to domain:
     - stateful: true
 
 {%- if join_domain.admins %}
-{%- set admins = join_domain.admins|string|replace('[','')|replace(']','') %}
+{%- set admins = join_domain.admins|yaml|string|replace('[','')|replace(']','') %}
```
Comment (lorengordon) [inline join-domain/windows/init.sls]: ahhhh, i honestly can't believe i did this once upon a time. not sure what i was thinking. can you test something like

```
{%- set admins = join_domain.admins | join(',') %}
```

Comment (eemperor) [reply join-domain/windows/init.sls]: Once upon you equals me today.  😀  Read up a bit and I see the subtle difference which helps Salt to process it correctly. 

I tested this locally and it works like a champ.

Comment (eemperor) [reply join-domain/windows/init.sls]: @lorengordon this should be good to go.  change was verified on Windows with Salt 2016.11 and 2018.3.3

**Pull request merged**